client/wp-admin-scripts/print-shipping-label-banner/dismiss-modal/test/index.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * External dependencies
+ */
+import { shallow } from 'enzyme';
+import { Button } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import DismissModal from '../index.js';
+
+describe( 'Tracking events in DismissModal', () => {
+	const trackBannerEvent = jest.fn();
+
+	let dismissModalWrapper;
+
+	beforeEach( () => {
+		dismissModalWrapper = shallow(
+			<DismissModal
+				visible={ jest.fn() }
```

Comment:
`visible` is meant to be treated as a boolean param, so `true` probably makes more sense for the value.

Reply:
Updated